tests/common.go

```diff
@@ -5284,3 +5284,24 @@ func GetSubsetOfSlice[T any](items []T, length int) ([]T, error) {
 	}
 	return randomItems, nil
 }
+
+// RandomStrings generates a slice of n unique random strings of length characters.
+// The strings consist of lowercase letters a-z.
+func RandomStrings(n, length int) []string {
```

Comment:
We do not want to return a slice of string. This function should only return a single random string.
It should take the length of the string and return a string and error.

Reply:
Modified RandomStrings to return a single random string.